internal/pkg/mapper/orchestrator/sort.go

```diff
@@ -1 +1,99 @@
 package orchestrator
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"v.io/x/lib/toposort"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils"
+)
+
+type phasesSorter struct {
+	phasesKeys         []string                // all keys in the order as defined in the remote / local state
```

Comment:
`key` je:
- pri remote load ID z JSONu, prevedene do `string`
- pri local load je to relativna path ku `phase`.

Tu mame `slice` klucov v takom poradi, ako sme ich nacitali z API/disku:
- je to preto, ze ak maju nejake `phases` rovnake/ziadne zavislosti, tak sa toto poradie zachova
- napr. ako to vidi user v UI, take bude aj poradie adresarov
- ... a `map` neuchovava poradie klucov